cli/commands.go

```diff
@@ -57,6 +57,12 @@ func Commands(metaPtr *meta.Meta) map[string]cli.CommandFactory {
 			}, nil
 		},
 
+		"cors": func() (cli.Command, error) {
```

Comment:
I need to ask internally but I think we will not want to add a `cors` command. There are already too many CLI commands as-is and this is such an advanced feature that requiring using the API for it seems reasonable. Don't blow it away before I check into it though.